src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -1071,6 +1059,35 @@ private TransformEngineChain createTransformEngineChain()
             engineChain.add(mediaStreamTrackReceiver);
         }
 
+        // Padding termination.
+        PaddingTermination paddingTermination = getPaddingTermination();
+        if (paddingTermination != null)
+        {
+            engineChain.add(paddingTermination);
+        }
+
+        // RTX
+        RtxTransformer rtxTransformer = getRtxTransformer();
+        if (rtxTransformer != null)
+        {
+            engineChain.add(rtxTransformer);
+        }
+
+        // TODO RTCP termination should end up here.
+
+        RemoteBitrateEstimatorSingleStream
+            remoteBitrateEstimator = getRemoteBitrateEstimator();
+        if (remoteBitrateEstimator != null)
+        {
+            engineChain.add(remoteBitrateEstimator);
```

Comment:
Except that I "forgot" to implement the reverseTransform method (badpokerface). Please don't merge yet.